tst/testspecial/run_all.sh

```diff
@@ -6,9 +6,20 @@ set -ex
 # (so for out-of-tree builds, builddir and not srcdir)
 GAPDIR=${GAPDIR:-../..}
 
-retvalue=0
 gap="$GAPDIR/bin/gap.sh"
-for gfile in *.g; do
+
+
+if ./run_gap.sh "${gap}" "support/check_bytes_per_variable.g" | grep "8"; then
```

Comment:
Alternatively could get rid of `check_bytes_per_variable.g` again and do this::
```suggestion
if "${gap}" -A -b -c 'QUIT_GAP(GAPInfo.BytesPerVariable - 8);'; then
```